modules/rn_3_11_0.adoc

```diff
@@ -35,11 +35,24 @@ In {productname} 3.8, a new UI was introduced as a technology preview feature. W
 
 For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/use_red_hat_quay/index#selecting-dark-mode-ui[Selecting color theme preference on the {productname} v2 UI].
 
+[id="team-synchronization-oidc"]
+=== Team synchronization support via {productname} OIDC
+
+This release allows administrators to leverage an OpenID Connect (OIDC) identity provider to synchronization team, or group, settings, so long as their OIDC provider supports team synchronization. Administrators can easily apply repository permissions to sets of users without having to manually create and sync group definitions between {productname} and the OIDC group, which is not scalable.
```

Comment:
to synchronise groups so long as their OIDC provider supports retrieval of group information from ID token or `/userinfo` endpoint.

Reply:
Updated to "This release allows administrators to leverage an OpenID Connect (OIDC) identity provider to synchronization team, or group, settings, so long as their OIDC provider supports the retrieval of group information from ID token or the `/userinfo` endpoint."